src/pages/CustomApps/useApps.js

```diff
@@ -0,0 +1,67 @@
+import { useDataQuery, useConfig } from '@dhis2/app-runtime'
```

Comment:
Not for this change, but should we aim to consolidate the two different `useApps` versions at some point?  @mediremi

Reply:
:100: they do have very similar logic and it would be nice to have it all in one place